lib/flutie/page_title_helper.rb

```diff
@@ -3,11 +3,13 @@ module PageTitleHelper
   def page_title(options = {})
     page_title = Flutie::PageTitle.new(options)
 
-    Flutie::PageTitlePresenter.new(
-      page_title.app_name,
-      content_for(page_title.page_title_symbol),
-      page_title.separator,
-      options[:reverse]
-    ).to_s
+    escape_once(
+      Flutie::PageTitlePresenter.new(
+        page_title.app_name,
+        content_for(page_title.page_title_symbol),
+        page_title.separator,
+        options[:reverse],
+      ).to_s,
```

Comment:
@neilvcarvalho

> there shouldn't be any backwards compatibility issue in making `flutie` return HTML-safe strings

My first solution was calling `.html_safe` here and it worked, but only for the page title content. To make sure custom `separator` and custom `app_name` are also html-escaped, and once, I think we need to go with `escape_once`